src/backend/distributed/executor/multi_utility.c

```diff
@@ -133,6 +136,11 @@ static void ErrorIfDistributedAlterSeqOwnedBy(AlterSeqStmt *alterSeqStmt);
 static void ErrorIfUnsupportedTruncateStmt(TruncateStmt *truncateStatement);
 static bool OptionsSpecifyOwnedBy(List *optionList, Oid *ownedByTableId);
 static void ErrorIfDistributedRenameStmt(RenameStmt *renameStatement);
+static void ErrorIfUnsupportedAlterAddConstraintStmt(AlterTableStmt *alterTableStatement);
+static void ErrorIfNotSupportedForeignConstraint(Relation relation,
```

Comment:
I know this is existing function, but I realized we generally used "ErrorIf**Un**supported..." instead of "ErrorIf**Not**Supporter...", could you rename this and ErrorIfNotSupportedConstraint functions.